starlark/src/values/mod.rs

```diff
@@ -444,24 +444,40 @@ pub trait TypedValue {
     fn unfreeze_for_iteration(&mut self);
```

Comment:
Why not also a default for freeze_for_iteration/unfreeze_for_iteration?

Reply:
Because it is not safe to have a default empty implementation for mutable types.

In other words, the implementation should explicitly state that they do not need freezing.

Reply:
This function won't be a part of public API when https://github.com/google/starlark-rust/pull/82 is done BTW.

Reply:
Ok SGTM